libbeat/processors/annotate/kubernetes/kubernetes.go

```diff
@@ -52,18 +52,44 @@ func newKubernetesAnnotator(cfg common.Config) (processors.Processor, error) {
 		return nil, err
 	}
 
+	//Load default indexer configs
+	if config.DefaultIndexers.Enabled == true {
+		for key, cfg := range Indexing.defaultIndexerConfigs {
+			config.Indexers = append(config.Indexers, map[string]common.Config{key: cfg})
```

Comment:
that's as of today, some day we may add changes like processors reloading that could mess with concurrency. I prefer just to trust the functions are thread safe as other modules are calling them